### ISSUE TITLE
Replace Program Overview one-pager

### ISSUE THREAD

**Issue by kmonterr** [18F/nsf-sbir#1115]
[overview one-pager 2018.pdf](https://github.com/18F/nsf-sbir/files/1916315/overview.one-pager.2018.pdf)
can you please name it overview2018

Hi Sarah,

Can you please replace the old one-pager with this one? I believe it is on the For Applicants Page and the For Press page. You might want to do a search too to make sure!

Thank you!!

**Comment by kmonterr** [comment]
[overview one-pager 2018.pdf](https://github.com/18F/nsf-sbir/files/1917172/overview.one-pager.2018.pdf)
Please use this version instead, @seakhtar ! Thank you!

**Comment by seakhtar** [comment]
Hi @kmonterr ,

I've uploaded the file (and made it 508-compliant, in case?) and have changed the two links to point to it- I couldn't find it in any other locations. I've also removed the old overview page off the server so no one stumbles over it when using the site search. 

I can go ahead and push to production, but there were some other changes to the webinar pages showing up as well and I didn't want to publish them without your ok. Let me know if I should go ahead.

Thanks!
